dask/highlevelgraph.py

```diff
@@ -59,6 +59,7 @@ class Layer(collections.abc.Mapping):
     annotations: Optional[Mapping[str, Any]]
 
     def __init__(self, annotations: Mapping[str, Any] = None):
+        self.info = {}
```

Comment:
1. please add type annotations above
2. please add documentation
3. you'll lose everything on clone(), cull(), and possibly some other methods

Reply:
1. please add type annotations above - added
2. please add documentation - added
3. you'll lose everything on clone(), cull(), and possibly some other methods - I think this is a comment, not a request to fix just now? If I've misunderstood please let me know.